keyboards/4x4/keymaps/default/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2018 REPLACE_WITH_YOUR_NAME
```

Comment:
Please change REPLACE_WITH_YOUR_NAME

Reply:
i'm just going to delete the whole block, if that's ok. i'd rather give the credit to @di0ib, who did all the work on this board already. I'm just porting his work. but i don't want to put his name, since he'd probably do a much  better job of this than me....
FWIW, on my other stuff, i put my name 😄